src/plugins/drawing-tool/model/drawing-basic-types.ts

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
I kept them separate due to the dependency graph. So currently:
drawing-types -> drawing-object-manager -> all objects (line, vector...) -> basic-drawing-types

If we combine them then it makes a cycle. It seems to work, but I thought it was better to avoid that.